modules/saml/www/idp/certs.php

```diff
@@ -18,19 +18,22 @@
 
 switch ($_SERVER['PATH_INFO']) {
     case '/new_idp.crt':
-        $certInfo = SimpleSAML\Utils\Crypto::loadPublicKey($idpmeta, false, 'new_');
```

Comment:
Doesn't make sense to _not_ require its existence and then display an undefined index error below.
Set false > true to require it to exist. That will also guarantee us it returns an array